### PR TITLE
add note about not patching common JS objects

### PR DESCRIPTION
**Pull request by sandlerr**
:koala:

/cc @zendesk/quokka

### CODE REVIEW

**Comment by gfxmonk** [comment]
:+1:

File: Bestpractices.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ If you aren't sure about any of the points below, if you have questions about th
 
 ```js
 (function() {
```
Comment (danielbreves) [inline Bestpractices.md]: Can we get rid of this section please?

**Comment by itirkarp** [comment]
:+1:

**Comment by danielbreves** [comment]
I really don't like this doc, it's so obnoxious. I know it isn't part of this PR, but could we change where we say "never" to "don't"?

**Comment by danielbreves** [comment]
> not patching common JS objects

@sandlerr when you say common JS did you mean native JS?

**Comment by sandlerr** [comment]
Not exactly. I think it would be bad for developers to patch any library we give them as well that isn't scoped to their app.

**Comment by danielbreves** [comment]
@sandlerr are you talking about common js modules owned by each app or framework modules?

**Comment by sandlerr** [comment]
I'm talking about two things here.

1. native JS objects
2. any JS objects provided by libraries that are not scoped to the app, such as moment and bootstrap

**Comment by danielbreves** [comment]
@sandlerr right, it's just the term "CommonJS" was a bit confusing in this context, thanks for clarifying.

**Comment by danielbreves** [comment]
@sandlerr thanks! :+1:

**Pull request merged**